ansible/roles/kong-api/defaults/main.yml

```diff
@@ -1067,7 +1081,7 @@ kong_apis:
 
   - name: freshDesk
     uris: "{{ tickets_prefix }}/v1/create"
-    upstream_url: "{{ freshDesk_url }}/api/v2/tickets"
+    upstream_url: "{{ freshDesk_url }}/v2/tickets"
```

Comment:
is this getting changed?

Reply:
it is not changing before itself it is mentioned as /v2/tickets" only,  so  that i replace it as same

Reply:
@sivakarthikeyanoptit Please keep it the same as the original one.

Reply:
@rjshrjndrn done